Documentation/concepts/observability/hubble-configuration.rst

```diff
@@ -127,69 +137,54 @@ it has the following advantages:
 
 **Troubleshooting**:
 
-If you get the following error while install cilium (and cert-manager), it's because
-cert-manager ValidatingWebhook (which used to verify the Certificate CRD resources)
-is not available (blocked due to CNI is not available).
+While installing Cilium or cert-manager you may get the following error:
 
 ::
 
     Error: Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": dial tcp x.x.x.x:443: connect: connection refused
 
-There are several ways to overcome above issue, such as:
+This happen when cert-manager's webhook (used to verify the ``Certificate`` CRD
+resources) is not available. There are several ways to workaround this issue:
 
-* Option 1: disable validation on cilium installed namespace
+* Option 1: install cert-manager CRDs before Cilium and cert-manager (see
```

Comment:
We could consider changing these options into a tabbed section like we have for the install guide.

Reply:
Make sense!